_posts/2021-03-12-automated-accessibility-testing-leveraging-github-actions-and-pa11y-ci-with-axe.md

```diff
@@ -195,9 +195,9 @@ As noted in our [playbook for automated testing](/playbook/automated-testing):
 * [Automated accessibility testing with Travis CI](https://andrewmee.com/posts/automated-accessibility-testing-node-travis-ci-pa11y/)
 * [How to Automate Web Accessibility Testing](https://medium.com/@f3igao/how-to-automate-web-accessibility-testing-921512bdd4bf)
 * Examples of applying the above steps to Jekyll sites:
-  * [BBC's "Accessibility, News and You"](https://github.com/bbc/accessibility-news-and-you) can be seen in this pull request: [https://github.com/dmundra/accessibility-news-and-you/pull/1](https://github.com/dmundra/accessibility-news-and-you/pull/1).
-  * [GSA's "Accessibility for Teams"](https://github.com/GSA/accessibility-for-teams) can be seen in this pull request: [https://github.com/dmundra/accessibility-for-teams/pull/1](https://github.com/dmundra/accessibility-for-teams/pull/1).
-  * [18F Accessibility Guide](https://github.com/18F/accessibility) can be seen in this pull request: [https://github.com/dmundra/accessibility-1/pull/1](https://github.com/dmundra/accessibility-1/pull/1).
+  * [BBC's "Accessibility, News and You"](https://github.com/bbc/accessibility-news-and-you) can be seen in [this pull request](https://github.com/dmundra/accessibility-news-and-you/pull/1).
+  * [GSA's "Accessibility for Teams"](https://github.com/GSA/accessibility-for-teams) can be seen in [this pull request](https://github.com/dmundra/accessibility-for-teams/pull/1).
```

Comment:
Ok, but lets make the link more descriptive. We shouldn't have 2 links called "pull request" going to different places. Imagine if we're listing this by links without the other context.

Reply:
Updated. How does that look?